grpcreflect_test.go

```diff
@@ -252,6 +260,7 @@ func testReflector(t *testing.T, reflector *Reflector, reflectionServiceFQN stri
 		}
 	})
 	t.Run("all_extension_numbers_of_type_missing", func(t *testing.T) {
+		t.Parallel()
```

Comment:
I think maybe we should just drop this linter and try our best to make the top-level tests parallel. What do you think?

Reply:
I don't mind making everything run in parallel but don't feel strongly about it. I doubt with a small project like this there is much of a perf benefit.